<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Взять почитать</string>
  </property>
  <widget class="QTableWidget" name="books_table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>581</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="order_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Взять выделенную книгу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанры</string>
   </property>
  </widget>
  <widget class="QComboBox" name="genres_filter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Примечание:&lt;br/&gt;В случае выделения нескольких книг,&lt;br/&gt;взята будет последняя из выделенных.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
